flake.nix

```diff
@@ -171,20 +171,11 @@
         myPackage = "hackage-mirror-tool";
         hsOverlay = pkgs: self: super:
           let
-            # FIXME: I think this can be removed now that we've upgraded to
```

Comment:
Sadly, this FIXME was wrong. Amazonka still hasn't had a release that pulls in the patch this revision includes. Without the patch (https://github.com/brendanhay/amazonka/pull/977), Amazonka is incompatible with Cloudflare R2, the object storage Stackage is using.

So, perhaps:

1. Add this pin back
2. Write a better FIXME based on what I just wrote :)

Reply:
97d9dd8e1549ef87184aada7adfe6db3182b4899 Added pin back and adjusted the fixme.